policy/resolver_test.go

```diff
@@ -265,12 +265,7 @@ policies:
 	})
 	require.NoError(t, err)
 	require.NotNil(t, rp)
-	require.Len(t, rp.CollectorJob.ReportingJobs, 2)
-	for _, rj := range rp.CollectorJob.ReportingJobs {
-		if rj.Type == policy.ReportingJob_CHECK {
-			require.Fail(t, "expected no check reporting job")
-		}
-	}
+	require.Len(t, rp.CollectorJob.ReportingJobs, 0)
```

Comment:
This seems to be an unintended consequence of this change. If you disable all the checks for a policy, it looks no different than disabling the policy

Reply:
I think it's fine to prune a policy from execution if no checks+queries are run, this is good behavior